course-v2/layouts/pages/section.html

```diff
@@ -1,3 +1,10 @@
 {{ define "main" }}
 {{ partial "course_content.html" . }}
+  {{ range .Pages }}
+  <div class="mb-2">
+    <article class="content pt-3 mt-1">
+      {{- .Content -}}
+    </article>
+  </div>  
```

Comment:
We should add tab indentation here: https://gohugo.io/functions/go-template/range/